### DISCUSSION TITLE
使用LLonebot一直403报错

### DISCUSSION THREAD

**Discussion by tianzt123456** [nonebot/discussions#522] [问答]
如题，.env的配置如下：
ENVIRONMENT=dev
DRIVER=~fastapi+~httpx+~websockets
ONEBOT_ACCESS_TOKEN=114514

LLonebot的配置如下：
{
  "satori": {
    "enable": true,
    "port": 5600,
    "listen": "0.0.0.0",
    "token": ""
  },
  "ob11": {
    "enable": true,
    "httpPort": 3000,
    "httpHosts": [],
    "httpSecret": "",
    "wsPort": 3001,
    "wsHosts": [
      "ws://127.0.0.1:8080/onebot/v11/ws"
    ],
    "enableHttp": true,
    "enableHttpPost": true,
    "enableWs": true,
    "enableWsReverse": true,
    "messagePostFormat": "array",
    "enableHttpHeart": false,
    "listenLocalhost": false
  },
  "heartInterval": 60000,
  "token": "114514",
  "enableLocalFile2Url": false,
  "debug": false,
  "log": false,
  "reportSelfMessage": false,
  "autoDeleteFile": false,
  "autoDeleteFileSecond": 60,
  "musicSignUrl": "",
  "msgCacheExpire": 120
}

bot日志如下：
10-08 19:42:51 [SUCCESS] nonebot | NoneBot is initializing...
10-08 19:42:51 [INFO] nonebot | Current Env: dev
10-08 19:42:51 [DEBUG] nonebot | Loaded Config: {'driver': '~fastapi+~httpx+~websockets', 'host': IPv4Address('127.0.0.1'), 'port': 8080, 'log_level': 'DEBUG', 'api_timeout': 30.0, 'superusers': set(), 'nickname': set(), 'command_start': {'/'}, 'command_sep': {'.'}, 'session_expire_timeout': datetime.timedelta(seconds=120), 'environment': 'dev', 'onebot_access_token': 114514}
10-08 19:42:52 [DEBUG] nonebot | Succeeded to load adapter "Console"
10-08 19:42:52 [SUCCESS] nonebot | Succeeded to load plugin "echo" from "nonebot.plugins.echo"
10-08 19:42:52 [SUCCESS] nonebot | Running NoneBot...
10-08 19:42:52 [DEBUG] nonebot | Loaded adapters: Console
10-08 19:42:52 [INFO] uvicorn | Started server process [7592]
10-08 19:42:52 [INFO] uvicorn | Waiting for application startup.
10-08 19:44:03 [SUCCESS] nonebot | Console unmounted.
10-08 19:44:03 [WARNING] nonebot | Press Ctrl-C for Application exit
10-08 19:44:03 [INFO] uvicorn | ('127.0.0.1', 58651) - "WebSocket /onebot/v11/ws" 403
10-08 19:44:03 [INFO] websockets | connection rejected (403 Forbidden)
10-08 19:44:03 [INFO] websockets | connection closed
10-08 19:44:06 [INFO] uvicorn | ('127.0.0.1', 58652) - "WebSocket /onebot/v11/ws" 403
10-08 19:44:06 [INFO] websockets | connection rejected (403 Forbidden)
10-08 19:44:06 [INFO] websockets | connection closed

求助！好人一生平安

**Comment by yanyongyu** [comment]
你没有安装加载 onebot 适配器

**Reply by tianzt123456** [reply]
感谢

**Reply by kamcdev** [reply]
> 你没有安装加载 onebot 适配器

我这里安装了也403
